meta/documents/user_guide_de.md

```diff
@@ -1,16 +1,16 @@
 # User Guide für das Plugin Ingenico ZVT
 
-Mit dem Plugin Ingenico ZVT verbinden Sie ein stationäres Terminal mit Drucker des Geräteherstellers Ingenico über das ZVT-Protokoll mit der Kassensoftware plentymarkets POS. 
+Mit dem Plugin Ingenico ZVT verbinden Sie ein stationäres Terminal mit Drucker des Geräteherstellers Ingenico über das ZVT-Protokoll mit der Kassensoftware plentymarkets POS.
 
 Um das Ingenico-Gerät mit plentymarkets POS nutzen zu können, benötigen Sie:
 
-* Ingenico Terminal mit Drucker, Betriebsart **ZVT**, Verbindungsart **TCP/IP**
+* Ingenico-Terminal mit Drucker, Betriebsart **ZVT**, Verbindungsart **TCP/IP**
 * Service-Vertrag mit einem Provider
 * plentymarkets POS Kasse
 * Mobiles Gerät, auf der die plentymarkets App installiert ist
 
 <div class="alert alert-warning" role="alert">
-   <strong><i>Hinweis:</strong></i> Um das Gerät mit plentymarkets POS zu verbinden, benötigen Sie die <b>Terminal-IP</b> und den <b>Port</b> des Ingenico Terminals. Diese Gerätedaten sind nur nach Eingabe des Techniker-Passworts auf dem Gerät zugänglich. Teilen Sie daher Ihrem Payment Provider rechtzeitig mit, dass Sie diese Daten benötigen, um das Terminal mit Ihrer Kassensoftware zu verbinden.
+   <strong>Um das Gerät mit plentymarkets POS zu verbinden, benötigen Sie die <b>Terminal-IP</b> und den <b>Port</b> des Ingenico Terminals. Diese Gerätedaten sind nur nach Eingabe des Techniker-Passworts auf dem Gerät zugänglich. Teilen Sie daher Ihrem Payment Provider rechtzeitig mit, dass Sie diese Daten benötigen, um das Terminal mit Ihrer Kassensoftware zu verbinden.
```

Comment:
Hier ist ein <strong> tag der nie geschlossen wird, alles danach wird in **bold** angezeigt